File: golang/basic/object.go
//Student object
package models

type Student struct{
	ID int
	FirstName string 
	LastName string
	Age int
}

func NewStudent(firstName string, lastName string, age int) *Student{
	return&Student{
		FirstName : firstName,
		LastName : lastName,
		Age: age,
	}
}

File: python/basic/object.py
import datetime

class Student():
    def __init__(self, id, firstname, lastname, age):
        self.id = id
        self.firstname = firstname
        self.lastname = lastname
        self.age = age
    
    def age1(self):
        self.age -= 1

        return self.age




def main():
   std = Student(1,"Nany", "Fonseca", 22)
   print(std.id)
   print(std.firstname)
   print(std.lastname)
   print(std.age)
   print(std.age1())
   print(std.age)
  


main()
